src/rust/lib/graph-editor/src/component/node.rs

```diff
@@ -379,13 +379,14 @@ impl NodeModel {
         let scene = scene.clone_ref();
         let input = InputEvents::new(&network);
 
-        let visualization_container = visualization::Container::new();
+        let visualization_container = visualization::Container::new(&scene);
         visualization_container.mod_position(|t| {
             t.x = 60.0;
             t.y = -120.0;
         });
 
         display_object.add_child(&visualization_container);
+        visualization_container.frp.set_visibility.emit(false);
```

Comment:
The container should not be visible by default. We should not need to emit event to hide it.